src/synth/drumModules/snareDrum.js

```diff
@@ -32,7 +32,7 @@ export default function (audioCtx, destination, time, { level, tone, snappy }) {
   outputVCA.amplitude.value = outputLevel;
 
   // envelopes
-  const noiseEnv = new ADGenerator(LINEAR, 0.1, 75, 0, 0.5);
+  const noiseEnv = new ADGenerator(LINEAR, 0.1, 160, 0, 0.9);
```

Comment:
In that case the lengthening of the snare's decay time should also go with a change to an `EXPONENTIAL` type AD envelope. In the case of the bridged-t osc I'm still experimenting with ways to emulate it but haven't had much luck so far.